core/Files.cc

```diff
@@ -203,4 +207,8 @@ string_view File::getLine(int i) {
     return source().substr(start, end - start);
 }
 
+std::string File::toString() const {
+    return path_;
+}
```

Comment:
If we already have to mention the name of the "show string for debugging purposes" method in the lldb file, i'd prefer to call this `.showRaw()`, and make it print

```
{ path = "<the path>" }
```

for consistency with our other `showRaw` methods (like `Loc`, `ast::Expression`, etc.)

`showRaw` always means "for debugging only" in Sorbet's codebase.

Reply:
I can make the script first look for a `showRaw` before attempting to use `toString`